mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLHostOps.td

```diff
@@ -34,13 +34,10 @@ def SYCLHostConstructorOp : SYCL_HostOp<"constructor"> {
     Operation representing the construction of a SYCL object, i.e., allocation,
     and member initialization.
 
-    This operation differs with `sycl.constructor` in two points:
-    - This operation will return a pointer to the constructed object instead of
-      receiving a reference to the object to construct
-    - This operation will return an `llvm.ptr` to any type instead of requiring a
-      `sycl` type.
+    This operation differs from `sycl.constructor` as it will return an 
```

Comment:
but it doesn't return anything, how about..
```suggestion
    This operation differs from `sycl.constructor` as it will construct a 
```

Reply:
True, I updated the wording.